### PR TITLE
chore: Updating Release workflow to include notes

### PR DESCRIPTION
**Pull request by rudouglas**
Will need to be reviewed and tested by DevEn team before merge

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -16,15 +16,28 @@ jobs:
...
+      last: ${{ steps.get-last-tag.outputs.last }}
+      notes: ${{ steps.get-release-notes.outputs.notes }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-
+      - name: Get last release
+        id: get-last-tag
+        run: |
+          LAST=$(gh pr list -B main -s merged --search "closed:>2021-02-28" -L 9999 | wc -l | awk '{print $1 - 1}')
```
Comment (zstix) [inline .github/workflows/release.yml:27]:
```suggestion
          LAST=$(gh release list -L 1 | awk '{print $1}')
```

**Comment by zstix** [comment]
dbd8ce3b3 Merge pull request #2393 from newrelic/edit-viz-in-dashboard
3569fd94c Merge pull request #2394 from newrelic/mmfred-patch-23
36af058fc Merge pull request #2376 from newrelic/DOC-6831---Improve-cancel/delete-accounts-docs
22a76259f Merge pull request #2391 from newrelic/2388-available-integrations
80c6940e5 Merge pull request #2390 from hredondo/develop
bddfb28e1 Merge pull request #2372 from newrelic/Doc4645-Unsupported_beans_types
9550bf57d Merge pull request #1 from hredondo/dashboards-migration-guide-fix-typos
162cc5048 Merge pull request #2389 from newrelic/Fixing-Browser-DT-link
883298073 Merge pull request #2359 from newrelic/clinton/bump-theme
57f1bb0d3 chore: merge latest in develop to resolve merge conflict yarn.lock
f2f7b9e65 Merge pull request #2373 from danybmx/remove-logging-dependencies

**Comment by zstix** [comment]
And here's a command to match the screenshot in the related issue:

```sh
git log release-158..release-159 --oneline --merges --format=format:'@%an: %s %h'
```

@Shawn Kilburn: Merge pull request #2393 from newrelic/edit-viz-in-dashboard dbd8ce3b3
@mmfred: Merge pull request #2394 from newrelic/mmfred-patch-23 3569fd94c
@zuluecho9: Merge pull request #2376 from newrelic/DOC-6831---Improve-cancel/delete-accounts-docs 36af058fc
@Anna Urbiztondo: Merge pull request #2391 from newrelic/2388-available-integrations 22a76259f
@Anna Urbiztondo: Merge pull request #2390 from hredondo/develop 80c6940e5
@Alba Olveros Bautista: Merge pull request #2372 from newrelic/Doc4645-Unsupported_beans_types bddfb28e1
@hredondo: Merge pull request #1 from hredondo/dashboards-migration-guide-fix-typos 9550bf57d
@Anna Urbiztondo: Merge pull request #2389 from newrelic/Fixing-Browser-DT-link 162cc5048
@Clinton: Merge pull request #2359 from newrelic/clinton/bump-theme 883298073
@Clinton: chore: merge latest in develop to resolve merge conflict yarn.lock 57f1bb0d3
@Shawn Kilburn: Merge pull request #2373 from danybmx/remove-logging-dependencies f2f7b9e65

**Comment by zstix** [comment]
Sorry for pasting that as plain text, I realized that that probably just pinged everyone in that comment 😨

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -16,15 +16,31 @@ jobs:
...
+      - name: Get last release
+        id: get-last-tag
+        run: |
+          LAST=$(gh release list -L 1 | awk '{print $1}')
+          echo "::set-output name=last::$LAST"
+          
+      - name: Get release notes
+        id: get-release-notes
+        run: |
+          NOTES=$(git log ${{ steps.get-last-tag.outputs.last }}..${{ steps.get-tag.outputs.tag }} --pretty="@%an - %b (%s)[%h]" --merges | sed -e 's!Merge pull request !!')
```
Comment (zstix) [inline .github/workflows/release.yml:40]: I think we want `..HEAD`. We haven't created the tag at this point (that happens in the next step) so `git` wont know what this reference is.

Comment (rudouglas) [inline .github/workflows/release.yml:40]:
```suggestion
          NOTES=$(git log ${{ steps.get-last-tag.outputs.last }}..HEAD --pretty="@%an - %b (%s)[%h]" --merges | sed -e 's!Merge pull request !!')
```

**Review by zstix — APPROVED** [review]

**Pull request merged**